README.md

```diff
@@ -44,6 +44,17 @@ run style checks use `composer cs`.
 
 ## Release notes
 
+### Version 5.0.0 (2017-03-29)
+
+#### Breaking changes
+
+* Added full text indices for donations and memberships. This breaks backward compatibility for MySQL versions < 5.6.
+* Prefixed all indices of donations with `d_`.
+
+#### Other changes
+
+* Updated `.travis.yml` to install MySQL 5.6.
```

Comment:
This is not something for the README as it's not relevant to people using this component